src/seedu/addressbook/Main.java

```diff
@@ -31,12 +31,13 @@
     private List<? extends ReadOnlyPerson> lastShownList = Collections.emptyList();
 
 
-    public static void main(String... launchArgs) {
+    public static void main(String... launchArgs) throws StorageOperationException {
         new Main().run(launchArgs);
```

Comment:
I thought main was the last caller. So why does it declaring throwing an exception? Shouldn't this be the place where all and any exceptions from the callers are handled?